crates/wysiwyg/Cargo.toml

```diff
@@ -10,13 +10,17 @@ name = "wysiwyg"
 version = "0.1.0"
 rust-version = "1.60"
 
-[features]
-
 [dependencies]
+bitflags = { version = "1.3.2", optional = true }
 html-escape = "0.2.11"
 html5ever = "0.25.2"
 once_cell = "1.13.0"
 widestring = "1.0.2"
 
 [dev-dependencies]
 speculoos = "0.9"
+pulldown-cmark = { version = "0.9.2", default-features = false }
+
+[features]
+default = ["to-markdown"]
+to-markdown = ["dep:bitflags"]
```

Comment:
I'm not sure I see the benefit of breaking out a to-markdown feature - we are always going to need it to send Matrix messages.

Reply:
I thought a feature was a good idea for the moment since it's still experimental. But I'm fine removing it. Tell me what you would prefer :-).

Reply:
Yeah, remove it please.